modules/ROOT/pages/proc_uprgading_pipelines_operator.adoc

```diff
@@ -0,0 +1,11 @@
+[id="installing-the-pipelines-operator_{context}"]
+= Upgrading the Pipelines Operator to the newer version
+
+OpenShift Pipelines upgrades are handled automatically via Operator Subscription. New updates will be installed as soon as they become available.
+
+[NOTE]
+====
+OpenShift Pipelines 0.5.2 has introduced changes incompatable with older versions. If the installed version is 0.5.0 or below, the operator *will not be automatically upgraded.*
```

Comment:
incompatable>incompatible
Remove the Bold format for 'will not be automatically upgraded.', since we are using the admonition (Note) to draw the user's attention, we do not need to use Bold in addition.